scripts/onboarding/aks/onboarding-msi-bicep-syslog/existingClusterOnboarding.bicep

```diff
@@ -89,9 +90,14 @@ resource aks_monitoring_msi_dcr 'Microsoft.Insights/dataCollectionRules@2022-06-
       ]
     }
     dataFlows: [
+      {
+        streams: defaultStreams
+        destinations: [
+          'ciworkspace'
+        ]
+      },
```

Comment:
remove the comma here otherwise it will throw 'Unexpected new line character after a comma' in the template